data/remnant missions.txt

```diff
@@ -1252,3 +1252,232 @@ mission "Remnant: Expanded Horizons Quarg 3"
+			`	"Oh, before I forget.." he hands you <payment>. "Off to start setting up the tests" he trundles off after the wagon disappearing down a distant corridor.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 4"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	to offer
+		has "Remnant: Broken Jump Drive 3: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`You head over to the Director's desk to see who is in need of the next broken jump drive. The director their smiles at you, looks over several messages on her screen, then clears her throat.`
+			`	"Your finds are very much in demand," she sings, "could you bring this one to a team working on <planet>?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Thank you. We are looking forward to seeing what comes from this."`
+				accept
+			label refuse
+			`	She taps something on her screen. "OK, I've notified them they will have to wait a bit. Let us know when you are ready."`
+				defer
+	on complete
+		conversation
+			`As you approach your designated pad you spot a team of researchers with unloading equipment hustling out of a nearby entrance. Once you have completed your shutdown sequence they quickly set to work unloading the drive. Once complete, they pause for a moment to hand you <payment>. "Thanks for finding this" they sing in unison. "Our labs are eager to begin work on your most recent find, Captain <first>. Our thanks!"`
+			choice
+				`	"How is your research going?"`
+					goto studying
+				`	"Good luck with that."`
+			label studying
+			`	"Good, and about to get better" they sing. "We are working on various aspects of a theory that the jump drive actually disregards traditional jumping entirely, and uses the folding mechanism to put the ship itself into a pocket, while somehow triggering an opening to said pocket in the target system. If this is true, it could be fairly dangerous to try activating these anywhere close to fragile things like cities or mountains or whatnot, which is why they put our lab on the far side of the planet from all such things. So we have a bit of a trip to get to our lab, but it should be exciting once we get there. Fare well!" With that, they head off towards a large vehicle nearby.`
```

Comment:
Add a comma after "better" but inside the quotes.

Reply:
Got it. Thanks!